server/config/topic_page.proto

```diff
@@ -20,14 +20,23 @@ package datacommons;
 // placeholder as a field in the ReplacementStrings interface in 
 // static/js/topic_page/string_utils.ts.
 
+message PlacesWithin {
+  // Place dcid of the parent.
+  string parent_dcid = 1;
+  // Type of child places within parent_dcid to support.
+  string children_type = 2;
+}
+
 message PageMetadata {
   string topic_id = 1;
   string topic_name = 2;
   // List of places supported by this config.
   repeated string place_dcid = 3;
-  // Contained places to load for each parent place type.
+  // List of places within a containing place to generate topics for.
+  repeated PlacesWithin places_within = 4;
+  // Contained places to load for each parent place type, for use in charts.
   // This can be overwritten by tiles.
-  map<string, string> contained_place_types = 4;
+  map<string, string> contained_place_types = 5;
```

Comment:
is this still needed now that there is the new PlacesWithin with a parent place and child type?

Reply:
yes, this specifies what type of child places to include in the charts, whereas PlacesWithin specifies which places to generate for. the intent was to support the use case where they are different sometimes (or we might not always use PlacesWithin to generate a list of places for the config).

Reply:
oh, I see, ok, makes sense!